plugins/modules/foreman_scc_account.py

```diff
@@ -119,6 +119,7 @@
 
 def main():
     module = KatelloEntityAnsibleModule(
+        entity_name='scc_account',
```

Comment:
Can't this be avoided if the class is `KatelloSccAccountModule`? I read the helper code that it should work?

https://github.com/theforeman/foreman-ansible-modules/blob/923f37f22af15f0b883e286fbb6a2ce1ce66a9c6/plugins/module_utils/foreman_helper.py#L770-L786

Reply:
Sure it can, but i ended up subclassing KatelloEntityAnsibleModule (3 loc) just to trigger that extra magic, while this is one concice line to describe the entity in question.
I prefer this one.

Reply:
But it makes it inconsistent with the foreman modules :(

Reply:
I won't block this, but highly prefer consistency over 2 lines save.